CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # CHANGELOG
 
-## v3.0.0 - Unreleased
+## v3.1.0 - Unreleased
```

Comment:
```suggestion
## Unreleased
```

Reply:
Changed `## v3.1.0 - Unreleased` to `## Unreleased`